components/sidebar_right/sidebar_right.tsx

```diff
@@ -255,14 +258,20 @@ export default class SidebarRight extends React.PureComponent<Props, State> {
 
         return (
             <>
-                <div className={'sidebar--right sidebar--right--width-holder'}/>
                 <div
+                    className={'sidebar--right sidebar--right--width-holder'}
+                    ref={this.sidebarRightWidthHolder}
```

Comment:
Are you meaning to use the same ref for both this div and the `Resizable` below?

Reply:
I tried to change the width of the `sidebarRightWidthHolder` by delivering a ref.

Reply:
I wanted to deliver the ref of `div` to `ResizableRhs` using `forwardRef`, so I thought that the element that ref points to is only `div` and both `refs` are necessary.

Reply:
Ooh okay, I see what the confusion is. I thought the `ref` prop of `ResizableRhs` meant that `sidebarRightWidthHolder` was holding a reference to the `ResizableRhs`, not that you were passing it into the `ResizableRhs` so that it would have access to the `div`.

Instead of having the prop to `ResizableRhs` being called `ref`, could you name it something like `containerRef`?

A prop called just `ref` generally means that the reference is pointing to that component. Similarly, you don't need `forwardRef` for `ResizbleRhs` because `forwardRef` is used to make the reference point to a child of that component rather than to the component itself. For example, we use `forwardRef` in [LocalizedInput](https://github.com/mattermost/mattermost-webapp/blob/master/components/localized_input/localized_input.tsx) so that someone passing a `ref` prop into it gets a reference to the `input` rather than the `Localizedinput` itself